packages/universal-router-sdk/src/utils/constants.ts

```diff
@@ -1,130 +1,88 @@
   },
   // arbitrum
   [ChainId.ARBITRUM_ONE]: {
     router: '0x0000000000000000000000000000000000000000',
-    weth: '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1',
-    creationBlock: 87206402,
     permit2Address: '0x0000000000000000000000000000000000000000',
   },
   // arbitrum goerli
   [ChainId.ARBITRUM_GOERLI]: {
     router: '0xa8EEA7aa6620712524d18D742821848e55E773B5',
-    weth: '0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3',
-    creationBlock: 18815277,
     permit2Address: '0xCaCC5DF33e3AF664158eaaB87bc8f282C674ec25',
   },
   // base goerli
   [ChainId.BASE_TESTNET]: {
     router: '0xa8EEA7aa6620712524d18D742821848e55E773B5',
-    weth: '0x4200000000000000000000000000000000000006',
-    creationBlock: 6915289,
     permit2Address: '0xCaCC5DF33e3AF664158eaaB87bc8f282C674ec25',
   },
   // base mainnet
   [ChainId.BASE]: {
     router: '0x0000000000000000000000000000000000000000',
-    weth: '0x4200000000000000000000000000000000000006',
-    creationBlock: 1452376,
     permit2Address: '0x0000000000000000000000000000000000000000',
   },
   [ChainId.POLYGON_ZKEVM]: {
     router: '0x0000000000000000000000000000000000000000',
-    weth: '0x0000000000000000000000000000000000000000',
-    creationBlock: 6915289,
     permit2Address: '0x0000000000000000000000000000000000000000',
   },
   [ChainId.POLYGON_ZKEVM_TESTNET]: {
     router: '0xa8EEA7aa6620712524d18D742821848e55E773B5',
-    weth: '0x30ec47F7DFae72eA79646e6cf64a8A7db538915b',
-    creationBlock: 6915289,
     permit2Address: '0xCaCC5DF33e3AF664158eaaB87bc8f282C674ec25',
   },
   [ChainId.LINEA]: {
     router: '0x0000000000000000000000000000000000000000',
-    weth: '0x0000000000000000000000000000000000000000',
-    creationBlock: 6915289,
     permit2Address: '0x0000000000000000000000000000000000000000',
   },
   [ChainId.LINEA_TESTNET]: {
     router: '0x9f3Cb8251492a069dBF0634C24e9De305d6946B8',
-    weth: '0x2C1b868d6596a18e32E61B901E4060C872647b6C',
-    creationBlock: 6915289,
     permit2Address: '0xCeccEEA1ee8f7420aB5A0daFA56faF39Ee794933',
   },
   [ChainId.OPBNB_TESTNET]: {
     router: '0xa8EEA7aa6620712524d18D742821848e55E773B5',
-    weth: '0x4200000000000000000000000000000000000006',
-    creationBlock: 6915289,
     permit2Address: '0xCaCC5DF33e3AF664158eaaB87bc8f282C674ec25',
   },
   [ChainId.ZKSYNC]: {
     router: '0x0000000000000000000000000000000000000000',
-    weth: '0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91',
-    creationBlock: 6915289,
     permit2Address: '0x0000000000000000000000000000000000000000',
   },
   [ChainId.ZKSYNC_TESTNET]: {
     router: '0x0000000000000000000000000000000000000000',
-    weth: '0x02968DB286f24cB18bB5b24903eC8eBFAcf591C0',
-    creationBlock: 6915289,
     permit2Address: '0x0000000000000000000000000000000000000000',
   },
 }
 
-export const UNIVERSAL_ROUTER_ADDRESS = (chainId: number): Address => {
+export const getUniversalRouterAddress = (chainId: ChainId): Address => {
```

Comment:
Also open to discussion here, what if we make it a safe get instead of throwing error?
```suggestion
export const getUniversalRouterAddress = (chainId: ChainId): Address | undefined => {
```

Reason for this suggestion is that, normally we don't expect a get address function to throw error, i.e. we usually don't have error catching calling this function

Reply:
i coukd change this to a swtich instead, that way we can have a default. say chain56 address. see my commetn way above

Reply:
yes perfect changing now